packages/@uppy/companion/src/server/provider/webdav/index.js

```diff
@@ -0,0 +1,80 @@
+
+const { validateURL } = require('../../helpers/request')
+const WebdavProvider = require('./common')
+const { ProviderUserError } = require('../error')
+const logger = require('../../logger')
+
+const defaultDirectory = '/'
+
+/**
+ * Adapter for WebDAV servers that support simple auth (non-OAuth).
+ */
+class WebdavSimpleAuthProvider extends WebdavProvider {
```

Comment:
Can we just make it a single class? Seems unnecessary to have a new base class for only one provider.